src/windows/2004/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# escape=`
+FROM mcr.microsoft.com/windows:2004
```

Comment:
You've confirmed that `mcr.microsoft.com/windows/servercore:2004` doesn't meet your needs?  It's smaller and preferable if it works.

Reply:
VS Build Tools fails to install with an inscrutable error in hcsshim::ImportLayer when I use servercore as the base